src/main/java/org/opentripplanner/routing/core/TraversalRequirements.java

```diff
@@ -15,11 +15,6 @@
      */
     public TraverseModeSet modes = TraverseModeSet.allModes();
 
-    /**
```

Comment:
What is this class used for? How is it different from just a RoutingRequest?

Reply:
It's used during P+R creation in `OpenStreetMapModule` to test that a P+R has a car entrance and a walkable exit.